### ISSUE TITLE
wfdb.rdsamp('mitdb/100', sampto=2000, pbdl=1) returns UnicodeEncodeError

### ISSUE THREAD

**Issue by tompollard** [MIT-LCP/wfdb-python#24]
```{python}
In [1]: import wfdb
```

```{python}
In [2]: sig, fields = wfdb.rdsamp('mitdb/100', sampto=2000, pbdl=1)
```

```{bash}
Downloading missing file(s) into directory: /Users/tompollard/sand
---------------------------------------------------------------------------
UnicodeEncodeError                        Traceback (most recent call last)
<ipython-input-2-bf902e201277> in <module>()
----> 1 sig, fields = wfdb.rdsamp('mitdb/100', sampto=2000, pbdl=1)

/usr/local/lib/python2.7/site-packages/wfdb/_rdsamp.pyc in rdsamp(recordname, sampfrom, sampto, channels, physical, stacksegments, pbdl, dldir, keepfiles)
   1029         sys.exit("input channels must be non-negative")
   1030 
-> 1031     dirname, baserecordname, filestoremove = checkrecordfiles(recordname, pbdl, dldir, keepfiles)
   1032 
   1033     fields = readheader(os.path.join(dirname, baserecordname))

/usr/local/lib/python2.7/site-packages/wfdb/_rdsamp.pyc in checkrecordfiles(recordname, pbdl, dldir, keepfiles)
    968     # Download physiobank files if specified
    969     if pbdl == 1:
--> 970         dledfiles = dlrecordfiles(recordname, dldir)
    971         if keepfiles==0:
    972             filestoremove = dledfiles

/usr/local/lib/python2.7/site-packages/wfdb/_rdsamp.pyc in dlrecordfiles(pbrecname, targetdir)
     45         for f in set(fields["filename"]):
     46             # Missing dat file
---> 47             dledfiles, displaydlmsg = dlifmissing(physioneturl+pbdir+"/"+f, os.path.join(targetdir, f), dledfiles, displaydlmsg, targetdir)
     48     else:  # Multi segment. Check for all segment headers and their dat files
     49         for segment in fields["filename"]:

/usr/local/lib/python2.7/site-packages/wfdb/_rdsamp.pyc in dlifmissing(url, filename, dledfiles, displaydlmsg, targetdir)
     81             print("File "+filename+" is already present.")
     82     else:
---> 83         dledfiles.append(dlorexit(url, filename, displaydlmsg, targetdir))
     84         displaydlmsg=0
     85 

/usr/local/lib/python2.7/site-packages/wfdb/_rdsamp.pyc in dlorexit(url, filename, displaydlmsg, targetdir)
     95         r = requests.get(url)
     96         with open(filename, "w") as text_file:
---> 97             text_file.write(r.text)
     98         return filename
     99     except requests.HTTPError:

UnicodeEncodeError: 'ascii' codec can't encode character u'\ufffd' in position 0: ordinal not in range(128)
```

**Comment by alistairewj** [comment]
```python
import wfdb
sig, fields = wfdb.rdsamp('mitdb/100', sampto=2000, pbdl=1)
```

```
Downloading missing file(s) into directory: /home/alistairewj/git/wfdb-python
Downloaded all missing files for record.
```

OS X specific?

**Comment by tompollard** [comment]
See: http://stackoverflow.com/questions/9942594/unicodeencodeerror-ascii-codec-cant-encode-character-u-xa0-in-position-20

```{python}
a = u'bats\u00E0'
print a
 => batsà
```

```{python}
str(a)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe0' in position 4: ordinal not in range(128)
```

**Comment by tompollard** [comment]
Ignore this...old issues that appears to have been fixed in 0.1.2!
